### PR TITLE
Remove old custom plugins and code that is needed to build them.

### PR DESCRIPTION
**Pull request by feanil**
We don't use the feature that we added to our fork.

@edx/devops @edx/testeng

### CODE REVIEW

File: playbooks/roles/jenkins_master/defaults/main.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ jenkins_group: "edx"
 jenkins_server_name: "jenkins.testeng.edx.org"
 jenkins_port: 8080
 
-jenkins_version: 1.574
+jenkins_version: "1.630"
```
Comment (benpatterson) [inline playbooks/roles/jenkins_master/defaults/main.yml:7]: Ooo nice.

File: playbooks/roles/jenkins_master/meta/main.yml [hunk 2]
```diff
@@ -1,3 +1,4 @@
 ---
 dependencies:
   - common
+  - oraclejdk
```
Comment (benpatterson) [inline playbooks/roles/jenkins_master/meta/main.yml:4]: I just want to call out that you may need to override defaults here if you want Java 8.

Comment (feanil) [inline playbooks/roles/jenkins_master/meta/main.yml:4]: It currently defaults to 7 and I thought that was fine for now.  Changing it to 8 seemed risky as the default.

**Comment by benpatterson** [comment]
You'll just want to be sure you are good with Java and from my perspective this gets a :+1:

File: playbooks/roles/jenkins_master/meta/main.yml (hunk 2 above)
Comment (benpatterson) [inline playbooks/roles/jenkins_master/meta/main.yml:4]: Ok. No objection here. Just the fact that you looked is all I wanted. We are planning to move to 8 in a month or so and we can revisit at that time.

**Comment by jibsheet** [comment]
:+1:

**Pull request merged**